src/js/fable-core/Array2.fs

```diff
@@ -1,16 +1,732 @@
 module Array
 
+// This value is for use by compiled F# code and should not be used directly.
+// We can use LanguagePrimitives.ErrorStrings.
+// TODO: Approve please if we really need to use ErrorStrings here
+#nowarn "1204"
```

Comment:
This is fine. Yes, the "value is unused" warning is annoying when using emits :wink: Another option is to prefix the arguments with an underscore, but the `#nowarn` directive is absolutely ok :+1:

Reply:
Thanks for the tip with underscores! Will do it, `Emit`s are going to be a less common citizen in the module.

As for the FS1204, do we need `LanguagePrimitives.ErrorStrings` as in the following example:
```f#
    if array.Length = 0 then invalidArg "array" LanguagePrimitives.ErrorStrings.InputArrayEmptyString
```
(The code is borrowed from FSharp.Core sources). Or is Fable compiler message localization going to be a whole different story and we should stick with hardcoded strings for now?

Reply:
Oh, I didn't know we could use `LanguagePrimitives.ErrorStrings` 😮 I've using hardcoded strings in JS but it makes sense to use the error strings here in F#, thanks for the tip!